treeopt/simulate.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import subprocess
+
+
+def execute_python_function(fun, x, args):
```

Comment:
I thought, that it would be wise to define functions to execute programs. But now I think, that there are so many possibilities for someone to call a program, that this way is not purposeful. That is why I changed it to a more general way (see example 02). The functions in treeopt.simulate are now not needed anymore and therefore have been removed.